content/docs/getting-started-tutorial/set-up.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Getting set up
+subtitle: Learn how to create a new Bigquery project and generate warehouse credentials.
+priority: 0
+---
+
+Welcome to the Dataform Getting Started Tutorial. This tutorial is for people who are new to Dataform and want to be taught how to set up a new project. We will show you how to create your own data model, how to test and document it and how to run schedules on it. You will need a working knowledge of SQL for this tutorial.
+
+For this tutorial we’re going to pretend we are a fictional e-commerce shop. We already have 3 main data sources in our data warehouse:
+
+- Information about our customers coming from Salesforce
+- Orders information from Shopify
+- Payment information from Stripe
+
+The aim of this tutorial is to create two new tables in our warehouse one called `order_stats` and one called `customers`, which are:
+
+- Updated every hour
+- Tested for data quality
+- Well documented
+
+<div style="position: relative; padding-bottom: 55.78124999999999%; height: 0;"><iframe src="https://www.loom.com/embed/2368b67928ec43b2a7eaf8fabda636f9" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
+
+## Create a new Bigquery project
+
+Dataform connects to [many different warehouses](https://docs.dataform.co/warehouses#__next) but for this tutorial we’ll use Bigquery, since anyone with a Google Account can use it and it has a free tier. We’ve created a public dataset in Bigquery that anyone can access for the purpose of this tutorial.
```

Comment:
```suggestion
Dataform connects to [many different warehouses](https://docs.dataform.co/warehouses) but for this tutorial we’ll use BigQuery, since anyone with a Google Account can use it and it has a free tier. We’ve created a public dataset in BigQuery that anyone can access for the purpose of this tutorial.
```